engine/src/main/java/org/terasology/logic/common/ActivateEvent.java

```diff
@@ -84,16 +84,24 @@ public int getActivationId() {
 
     public Vector3f getTargetLocation() {
         LocationComponent loc = target.getComponent(LocationComponent.class);
-        if (loc != null&& !Float.isNaN(loc.getWorldPosition().x)) {
```

Comment:
Do we know why this originally only checked x for being finite?

Reply:
An assumption that if x is nan then the whole vector is nan. Method from joml does not use this assumption.